.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -9,13 +9,17 @@ inputs:
     default: "false"
     required: false
 
-  image-name:
-    description: The name of the image, should match the repository name in ECR
-    required: true
-
-  ecr-registry:
-    description: The ECR registry to push to, used in docker/login-action and for tagging images
-    default: public.ecr.aws/chainlink
+  ecr-prefix:
+    description: The ECR registry scope
+    default: public.ecr.aws
+    required: false
+  ecr-repo-path:
+    description: Name of the ECR registry to push to, used in docker/login-action and for tagging images
+    default: chainlink/chainlink
+    required: false
+  aws-use-access-key:
```

Comment:
This seems redundant, we can check if `aws-access-key-id` is set to `false` or not instead

Reply:
Removed altogether per your next suggestion 🙂